pipelines/azure-pipelines-pr.yaml

```
@@ -30,6 +30,10 @@ stages:
         - checkout: self
           displayName: 'Checkout Module'
           fetchDepth: 1
+        - task: TerraformInstaller@2
+          displayName: 'Terraform Install'
+          inputs:
+            terraformVersion: 'latest'          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)